app/Http/Controllers/License/LicenseController.php

```diff
@@ -205,17 +204,23 @@ public function updateLicensedDomain($licenseCode, $domain, $productId, $license
      */
     protected function getIpAndDomain($domain)
     {
+
         $isIP = (bool) ip2long($domain);
-        if ($isIP == true) {
-            $requiredomain = 0;
-            $ip = $domain;
-            $domain = '';
+        if ($domain != '') {
+            if ($isIP == true) {
```

Comment:
== true isn't needed

Reply:
please follow camelCase convention for naming php variables. (requiredomain => requireDomain)